app.js

```diff
@@ -13,3 +13,14 @@ const navSlide = () => {
 };
 
 navSlide();
+const modalBtn = document.querySelector('.project-button');
+
```

Comment:
- [ ] Based on project requirements, you should have a JS object with all the project data that populates the popup modal:

![image](https://user-images.githubusercontent.com/77038610/151344337-75e41966-a72f-46d6-94be-809f2e0bb9ec.png)
 
Please add the JS object with project data.